### PR TITLE
Issue31 mock tests

### PR DESCRIPTION
**Pull request by missaugustina**
Resolves Issue #31

### CODE REVIEW

**Comment by missaugustina** [comment]
Just added a bunch of folks to review, but I don't need *all* of them ;)

**Comment by missaugustina** [comment]
Found a couple lines with debug code, I'll push an update to remove it.

**Comment by missaugustina** [comment]
just double checked and this is up to date! let me know what next steps you all recommend.

File: DESCRIPTION [hunk 1]
```diff
@@ -8,7 +8,8 @@ Authors@R: c(person("Gabriela", "De Queiroz", email = "gabriela@rladies.org", ro
              person("Olga", "Mierzwa-Sulima", email = "olga@rladies.org", role = c("aut")),
              person("Lucy", "D'Agostino McGowan", email = "lucy@rladies.org", role = c("aut")),
              person("Claudia", "Vitolo", role = c("aut")),
-             person("Michael", "Beigelmacher", role = c("ctb")))
+             person("Michael", "Beigelmacher", role = c("ctb")),
```
Comment (gdequeiroz) [inline DESCRIPTION]: Who is Michael Beigelmacher?

Comment (gdequeiroz) [reply DESCRIPTION]: nvm,. found it

**Comment by gdequeiroz** [comment]
It looks good to me.

**Pull request merged**